language/golang/language.go

```diff
@@ -46,7 +46,7 @@ func (l *Language) ImportPath(projectRootPath string, filePath string) (importPa
 
 // TestFilePath returns the file path of a test file given the corresponding file path of the test's source file.
 func (l *Language) TestFilePath(projectRootPath string, filePath string) (testFilePath string) {
-	return strings.TrimSuffix(filePath, l.DefaultFileExtension()) + l.DefaultTestFileSuffix()
+	return filepath.Join(projectRootPath, strings.TrimSuffix(filePath, l.DefaultFileExtension())+l.DefaultTestFileSuffix())
```

Comment:
I don't think this commit should happen. This method is based on the one we have internally at Symflower. The idea is to at one point have this not be just static but do the file dependent on the implementation and then on the configuration / type-of-test. The path is then still returned relative, because it should not matter for the basolution location of the repository, because that could be a copy. So only relative counts.